web_src/js/components/RepoActionView.vue

```diff
@@ -200,6 +200,12 @@ const sfc = {
       return await resp.json();
     },
 
+    async deleteArtifact(name) {
+      if (!window.confirm(this.locale.areYouSure)) return;
```

Comment:
Hmm…
Can we please conform with the rest of the application?
Typically we use a custom modal for this, not a window prompt.

Reply:
I test same case in github. It provides a simple confirm prompt currently. I think modal is heavy in this case.